t/plugin/opentelemetry.t

```diff
@@ -95,113 +77,34 @@ __DATA__
             if code >= 300 then
                 ngx.status = code
             end
-            ngx.say(body)
+            ngx.say("done")
         }
     }
-
-
-
-=== TEST 2: trigger opentelemetry
 --- request
-GET /opentracing
+GET /t
 --- response_body
-opentracing
---- wait: 1
---- grep_error_log eval
-qr/opentelemetry export span/
---- grep_error_log_out
-opentelemetry export span
+done
 
 
 
-=== TEST 3: use default always_off sampler
---- config
-    location /t {
-        content_by_lua_block {
-            local t = require("lib.test_admin").test
-            local code, body = t('/apisix/admin/routes/1',
-                ngx.HTTP_PUT,
-                [[{
-                    "plugins": {
-                        "opentelemetry": {
-                        }
-                    },
-                    "upstream": {
-                        "nodes": {
-                            "127.0.0.1:1980": 1
-                        },
-                        "type": "roundrobin"
-                    },
-                    "uri": "/opentracing"
-                }]]
-                )
-
-            if code >= 300 then
-                ngx.status = code
-            end
-            ngx.say(body)
-        }
-    }
-
-
-
-=== TEST 4: not trigger opentelemetry
+=== TEST 2: trigger opentelemetry
 --- request
 GET /opentracing
+--- wait: 2
 --- response_body
 opentracing
---- grep_error_log eval
-qr/opentelemetry export span/
---- grep_error_log_out
 
 
 
-=== TEST 5: use trace_id_ratio sampler, default fraction = 0
---- config
-    location /t {
-        content_by_lua_block {
-            local t = require("lib.test_admin").test
-            local code, body = t('/apisix/admin/routes/1',
-                ngx.HTTP_PUT,
-                [[{
-                    "plugins": {
-                        "opentelemetry": {
-                            "sampler": {
-                                "name": "trace_id_ratio"
-                            }
-                        }
-                    },
-                    "upstream": {
-                        "nodes": {
-                            "127.0.0.1:1980": 1
-                        },
-                        "type": "roundrobin"
-                    },
-                    "uri": "/opentracing"
-                }]]
-                )
-
-            if code >= 300 then
-                ngx.status = code
-            end
-            ngx.say(body)
-        }
-    }
-
-
-
-=== TEST 6: not trigger opentelemetry
---- request
-GET /opentracing
---- response_body
-opentracing
---- grep_error_log eval
-qr/opentelemetry export span/
---- grep_error_log_out
+=== TEST 3: check log
+--- exec
+tail -n 1 ci/pod/otelcol-contrib/data-otlp.json
+--- response_body eval
+qr/.*opentelemetry-lua.*/
 
 
```

Comment:
Are these two configurations equivalent?
```
                        "opentelemetry": {
                            "sampler": {
                                "name": "always_on"
                            }
                        }
```

```josn
                    "plugins": {
                        "opentelemetry": {
                        }
                    },
```

Reply:
no. The second one turns off the opentelemetry